src/HomieNode.hpp

```diff
@@ -15,12 +15,20 @@ class HomieNode {
   friend HomieInternals::HomieClass;
   friend HomieInternals::BootNormal;
   friend HomieInternals::BootConfig;
-  public:
+
+  virtual bool InputHandler(String property, String value);
+  virtual void loop();
+  virtual void setup();
+
+protected:
+  bool _subscribeToAll;
```

Comment:
it was necessary to make it protected (instead of private) to change it in constructor of derived Nodes.